Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -396,9 +396,8 @@ class HistoryPanel: SiteTableViewController, HomePanel {
     }
 
     func showSyncedTabs() {
-        let nextController = RemoteTabsPanel()
+        let nextController = RemoteTabsPanel(profile: profile)
```

Comment:
Cleanup item: remove need for `var profile:Profile!`
I can't help myself! I must fix this when I touch existing code.